taker-frontend/src/App.tsx

```diff
@@ -42,6 +44,22 @@ async function postCfdOrderRequest(payload: CfdOrderRequestPayload) {
 export const App = () => {
     const toast = useToast();
 
+    const {
+        lastMessage,
+        readyState,
+    } = useWebSocket("wss://www.bitmex.com/realtime?subscribe=instrument:.BXBT", {
```

Comment:
We should do this in the daemon. After discussion with @itchymax we realized that the daemon price feed might not be representing all prices we need. 
The way we internally use the price is not correct in all cases. We should properly clean this up, but this might take more effort.

If we decide to do this in a follow up, please create a ticket that describes the motivation of the different prices and which one to use when.

Reply:
Issue created #533